package.json

```diff
@@ -2,35 +2,46 @@
   "name": "webpack-retry-chunk-load-plugin",
   "version": "2.2.0",
   "description": "A webpack plugin to retry loading of chunks that failed to load",
-  "main": "src/index.js",
+  "main": "build/src/index.js",
```

Comment:
We should also add a `types` entry here that points to the declaration file so people can benefit from the typings

Reply:
indeed! done